lib/AST/ASTMangler.cpp

```diff
@@ -1379,8 +1379,12 @@ void ASTMangler::appendType(Type type, GenericSignature sig,
     case TypeKind::BuiltinUnsafeValueBuffer:
       return appendOperator("BB");
     case TypeKind::BuiltinUnboundGeneric:
-    case TypeKind::Locatable:
-      llvm_unreachable("not a real type");
+      llvm::errs() << "Don't know how to mangle a BuiltinUnboundGenericType\n";
+      abort();
```

Comment:
Not going to hold up this patch over this, but we really need to convert this function to return `llvm::Error` at some point. These are landmines when the code is called from LLDB because it kills the debug session...

Reply:
Also, thanks!